fvm/src/executor/default.rs

```diff
@@ -104,16 +104,16 @@ where
                 }
             }
             Err(ExecutionError::OutOfGas) => Receipt {
-                exit_code: ExitCode::SysErrOutOfGas,
+                exit_code: SystemExitCode::OUT_OF_GAS,
                 return_data: Default::default(),
                 gas_used,
             },
             Err(ExecutionError::Syscall(err)) => {
                 let exit_code = match err.1 {
-                    ErrorNumber::IllegalOperation => ExitCode::SysErrIllegalActor,
-                    ErrorNumber::AssertionFailed => ExitCode::SysErrIllegalArgument,
-                    ErrorNumber::InsufficientFunds => ExitCode::SysErrInsufficientFunds,
-                    ErrorNumber::NotFound => ExitCode::SysErrInvalidReceiver,
+                    ErrorNumber::IllegalOperation => SystemExitCode::ILLEGAL_ACTOR,
+                    ErrorNumber::AssertionFailed => SystemExitCode::ASSERTION_FAILED,
+                    ErrorNumber::InsufficientFunds => SystemExitCode::INSUFFICIENT_FUNDS,
+                    ErrorNumber::NotFound => SystemExitCode::INVALID_RECEIVER,
                     code => {
                         return Err(anyhow!(
```

Comment:
Future comment: We should probably change this to return `SysErrAssertionFailed` rather than a fatal error.

Reply:
#438